Gruntfile.js

```diff
@@ -469,6 +475,48 @@ module.exports = function (grunt) {
                     compress: true
                 }
             }
+        },
+        verifyDistFiles: {
+            js: {
+                verifyFilesListFunc: function () {
+                    var expectedFilenames = [];
+                    var distributions = grunt.config.get("distributions");
+                    _.forEach(distributions, function (value, distribution) {
+                        var jsFilename = "infusion-" + distribution + ".js";
+                        var mapFilename = jsFilename + ".map";
+                        var distDirectory = "dist/";
+                        expectedFilenames.push(distDirectory + jsFilename, distDirectory + mapFilename);
+                    });
+                    return expectedFilenames;
+                }
+            },
+            css: {
+                files: [
+                    {
+                        expand: true,
+                        src: ["*.styl"],
+                        cwd: "src/framework/preferences/css/stylus/",
+                        dest: "dist/assets/src/framework/preferences/css/stylus/",
```

Comment:
There is also a stylus file from the Switch component that gets compiled and placed in the Assets directory. We should probably test for this one too.